README.md

```diff
@@ -9,3 +9,45 @@ Documentation
 -------------
 
 https://axom.readthedocs.io/en/master/
+
+Getting Involved
+----------------
+
+Axom is an open-source project and we welcome contributions from the community.
+
+Mailing List
+------------
+
+The project maintains two email lists: 
+
+  * 'axom-users@llnl.gov' is how Axom users can contact developers for questions, report issues, etc.
+  * 'axom-dev@llnl.gov' is for communication among team members.
+
+Contributions
+-------------
+
+We welcome all kinds of contributions: new features, bug fixes, documentation edits.
+
+To contribute, make a [pull request](https://github.com/llnl/axom/compare), with `develop` as the destination branch.
+We use CI testing and your branch must pass these tests before being merged.
+
+For more information, see the [contributing guide](https://github.com/llnl/axom/blob/develop/CONTRIBUTING.md).
+
+Authors
+-------
+
+Thanks to all of Axom's
+[contributors](https://github.com/llnl/axom/graphs/contributors).
+
+Release
+-------
+
+Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC. 
+Produced at the Lawrence Livermore National Laboratory.
+
+All rights reserved. See details in the file LICENSE.
```

Comment:
Would it make sense to remove the "It is also linked here" part, and just add a link the the word LICENSE?
E.g.
> All rights reserved. See details in the [LICENSE](./LICENSE) file.